### ISSUE TITLE
[Item] Mote of Shadow pre 2.1

### ISSUE THREAD

**Issue by Kidcudi123** [Atlantiss/NetherwingBugtracker#3348]
Current Behaviour: Only the Void/Wraith kind drops this item
**Expected Behaviour:** In our current patch - Mobs like Eye of Culuthas should drop MoS

![image](https://user-images.githubusercontent.com/33790123/49700858-94a8b600-fbe4-11e8-814c-e1b46fc533ca.png)

![image](https://user-images.githubusercontent.com/33790123/49700811-00d6ea00-fbe4-11e8-8e4d-9f2a870e3fb1.png)

Sources: 
http://wowwiki.wikia.com/wiki/Mote_of_Shadow
https://www.wowhead.com/npc=20394/eye-of-culuthas#comments

**Comment by Neophrone** [comment]
Yeah, I remember when I was playing retail, AH price was around 2-5g per 1 Primal Shadow, and after 2.1 patch it grew to 18-25. I was surprized, but then I found out what did change. 
Mostly every demon in Outland had Mote of Shadow(1-2) in it's drop list before 2.1 patch.
https://web.archive.org/web/20070208163610/http://www.wowhead.com:80/?item=22577#00cRz

**Comment by Manaci** [comment]
Irrelevant now

**Comment by Manaci** [comment]
Reopened for future usage

**Comment by bekonja** [comment]
https://web.archive.org/web/20070410052402/http://thottbot.com/i22577
full list of demons in "dropped by" tab
would be nice to have it fixed for fresh

**Comment by Godmatik** [comment]
Rev 4017 - confirmed
